resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterScheduler.scala

```diff
@@ -430,6 +429,9 @@ private[spark] class MesosClusterScheduler(
   }
 
   private def getDriverUris(desc: MesosDriverDescription): List[CommandInfo.URI] = {
+    val useFetchCache = conf.getBoolean("spark.mesos.fetcherCache.enable", false) ||
+      desc.conf.getBoolean("spark.mesos.fetcherCache.enable", false)
```

Comment:
Ur, this one looks the opposite direction. Could you check this again?

Reply:
Although this is effectively same, could you preserve the order of boolean expressions to be consistent?

Reply:
Done